source/usage-examples/replaceOne.txt

```diff
@@ -0,0 +1,46 @@
+==================
+Replace a Document
+==================
+
+.. default-domain:: mongodb
+
+You can replace a single document using the `replaceOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#replaceOne>`_
+method. ``replaceOne()`` accepts a query document and a replacement
+document, and replaces the first document that matches the filter with
```

Comment:
Nitpick: I wonder if it would be more clear to use either query or filter rather than interchangeably within the same section without explaining that they are equivalent since this could be new terminology to a Jasmine persona. Also make sure the term you choose syncs with the terminology/variable naming in your code snippet.

Reply:
Since I've stuck with query in the past, I'm going to use that same term everywhere. Good catch.